### PR TITLE
Implemented refined visibility modal design

### PR DESCRIPTION
**Pull request by bp323**
screen shot 2015-02-27 at 11 32 31

The original design can be found in Google Drive

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: apps/timetable/admin/ui/css/admin.css [hunk 1]
```diff
@@ -867,11 +857,21 @@
...
     position: absolute;
     right: 8px;
+    top: 6px;
+}
+
+#gh-visibility-modal #gh-visibility-form label .gh-visibility-icon-container {
+    -webkit-border-radius: 2px;
+       -moz-border-radius: 2px;
+            border-radius: 2px;
+    border: 3px solid;
```
Comment (Coenego) [inline apps/timetable/admin/ui/css/admin.css]: `border` should go before `border-radius`

File: apps/timetable/admin/ui/css/admin.css [hunk 2]
```diff
@@ -883,20 +883,53 @@
...
+}
+
+#gh-visibility-modal #gh-visibility-form #gh-visibility-divider {
+    font-size: 55px;
+    padding: 77px 0 0;
+}
+
+#gh-visibility-modal #gh-visibility-form #gh-visibility-divider i.fa {
+    margin: 0 auto;
+    display: block;
```
Comment (Coenego) [inline apps/timetable/admin/ui/css/admin.css]: `display` should go before `margin`

**Comment by Coenego** [comment]
Left some minor comments. Reassigning to @bp323

**Comment by bp323** [comment]
Back to @Coenego for review.

**Pull request merged**